### PR TITLE
new interactive tool for Qiskit in Galaxy

### PR DESCRIPTION
**Pull request by thepineapplepirate**
This is the XML file for the new Qiskit in Galaxy tool, for quantum computing with IBM quantum systems. It is built on the existing JupyTool XML file. The respository for the docker files can be found here https://github.com/thepineapplepirate/qiskit-jupyter-galaxy_docker.git

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: tools/interactive/interactivetool_qiskit_jupyter_notebook.xml [hunk 1]
```diff
@@ -0,0 +1,313 @@
+<tool id="interactive_tool_qiskit_jupyter_notebook" tool_type="interactive" name="Qiskit Jupyter notebook" version="0.0.1" profile="22.01">
+    <description>interactive tool</description>
+    <requirements>
+        <container type="docker">thepineapplepirate/qiskit_galaxy:latest</container>
```
Comment (bgruening) [inline tools/interactive/interactivetool_qiskit_jupyter_notebook.xml]: Can you please version this container?
Do you want to add a comment where the image can be found on github?

File: tools/interactive/interactivetool_qiskit_jupyter_notebook.xml [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+            <param name="ipynb" value="test.ipynb"/>
+            <param name="run_it" value="true"/>
+            <output name="jupyter_notebook" file="test.ipynb" ftype="ipynb"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+    Welcome to the **Qiskit JupyTool**! This tool incorporates the entire Qiskit stack, including tutorials and supplemental resources. This tool is an extension
+    of the standard **JupyTool**, so a full functionality of Jupyter lab is also included. You can create, run, and share custom Galaxy tools based upon Jupyter Notebooks.
+
+    Qiskit is a python based, open-source software stack for quantum computing on IBM quantum systems. When run, the Jupyter lab instance includes directories
```
Comment (bgruening) [inline tools/interactive/interactivetool_qiskit_jupyter_notebook.xml]:
```suggestion
    Qiskit is a Python based, open-source software stack for quantum computing on IBM quantum systems. When run, the Jupyter lab instance includes directories
```

File: tools/interactive/interactivetool_qiskit_jupyter_notebook.xml (hunk 1 above)
Comment (thepineapplepirate) [reply tools/interactive/interactivetool_qiskit_jupyter_notebook.xml]: ok will do, thank you sir!

Comment (thepineapplepirate) [reply tools/interactive/interactivetool_qiskit_jupyter_notebook.xml]: somehow things got messed up, will have to make a new PR or something.

**Pull request closed without merging**